client/blocks/product-plan-overlap-notices/index.jsx

```diff
@@ -95,6 +96,12 @@ class ProductPlanOverlapNotices extends Component {
 		return availableProducts[ currentPlanSlug ].product_name;
 	}
 
+	clickPurchaseHandler = purchaseId => {
+		this.props.recordTracksEvent( 'calypso_product_overlap_purchase_view', {
+			purchase_id: purchaseId,
```

Comment:
instead of passing the purchase_id. I think it would be more useful to pass in the purchase_slug. (the thing that identifies the product)
This way we can group the products together. Since the purchaseId is unique we would expect it to be the different across all the different events.

Reply:
Resolved in f323454bde2552d4cc774edfa53811b5207d2741.